src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -314,6 +314,12 @@ protected override void SetupEditedSpecies()
         base.SetupEditedSpecies();
     }
 
+    protected override void ApplyComponentLightLevels()
+    {
+        base.ApplyComponentLightLevels();
+        patchMapTab.UpdateShownPatchDetails();
```

Comment:
Patch map is an editor component? Why doesn't it get given the info through the editor component light level update callback?

Reply:
That didn't cross my mind...